spanner_orm/field.py

```diff
@@ -297,11 +319,11 @@ def field_type_from_ddl(ddl: str) -> FieldType:
     return Integer()
   elif ddl == 'FLOAT64':
     return Float()
-  elif ddl == 'STRING(MAX)':
+  elif re.fullmatch(r'STRING\((?:[0-9]+|MAX)\)', ddl) is not None:
     return String()
   elif ddl == 'TIMESTAMP':
     return Timestamp()
-  elif ddl == 'BYTES(MAX)':
+  elif re.fullmatch(r'BYTES\((?:[0-9]+|MAX)\)', ddl) is not None:
     return BytesBase64()
```

Comment:
These should both return a type with the correct length. E.g., 'STRING(5)' should be converted to `String(5)` instead of `String()`. See line 328-329 below for how to do that easily. (This should prevent bugs in code like https://github.com/google/python-spanner-orm/blob/fa9dead636bcf6f31a04965e4cf2d3ebfa7e91a2/spanner_orm/admin/update.py#L223 that would otherwise ignore a change from `STRING(5)` to `STRING(MAX)` and prevent a change from `STRING(5) NOT NULL` to `STRING(5)`.)

Reply:
I see. I've modified it so that String(5) will return String(5), same for bytes.